src/playground/components/AlertsActionBar.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+
+export default function AlertsActionBar({
+	messages,
+	onFixAll,
+	options,
+	onUpdate,
+	rulesWithInvalidConfigs,
+	setRulesWithInvalidConfigs,
+}) {
+	const hasFixableMessages = messages.some(message => message.fix);
+	const hasDisableableMessages = messages.some(
+		message => !message.suggestions && options.rules[message.ruleId],
+	);
+
+	let description = "";
+
+	if (hasFixableMessages && hasDisableableMessages) {
+		description =
+			"Resolve all fixable issues or disable their respective rules from config";
+	} else if (hasFixableMessages) {
+		description = "Resolve all fixable issues";
+	} else if (hasDisableableMessages) {
+		description =
+			"Disable all rules associated with these issues from config";
+	}
+
+	const handleDisableAll = () => {
+		const updatedOptions = { ...options };
+
+		messages.forEach(message => {
+			if (!message.suggestions) {
+				delete updatedOptions.rules[message.ruleId];
+				setRulesWithInvalidConfigs(
+					new Set(
+						[...rulesWithInvalidConfigs].filter(
+							rule => rule !== message.ruleId,
+						),
+					),
+				);
+			}
+		});
+		onUpdate(Object.assign({}, updatedOptions));
```

Comment:
```suggestion
		    onUpdate(updatedOptions);
```
Object.assign is unnecessary here as spread is already used